src/utils/__mocks__/worker-factory.js

```diff
@@ -3,18 +3,56 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // @flow
 
-import Worker from 'workerjs';
+// $FlowExpectError Flow doesn't know about this util
```

Comment:
If you're OK with this PR I'll probably implement a small libdef for this module.